server/jetstream.go

```diff
@@ -453,6 +454,7 @@ func (s *Server) enableJetStream(cfg JetStreamConfig) error {
 		s.Noticef("")
 	}
 	s.Noticef("---------------- JETSTREAM ----------------")
+	s.Noticef("  Strict:          %t", cfg.Strict)
```

Comment:
I think we can remove these from banner

Reply:
Let's only print it if `cfg.Strict == true`.

Reply:
This resolved?